bots/parsebot.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+from github import Github
+from pathlib import Path
+import subprocess
+import html2text
+import requests
+import shlex
+import json
+import time
+import re
+import os
+
+
+def scan(r):
```

Comment:
Could `r` be something more descriptive?

Reply:
Renamed to `session` in latest commit.